analyzer/tests/functional/analyze_and_parse/test_files/compiler_warning_no_warn.output

```diff
@@ -18,11 +18,40 @@ CHECK#CodeChecker check --build "make compiler_warning_simple" --output $OUTPUT$
 [] - To store results use the "CodeChecker store" command.
 [] - See --help and the user guide for further options about parsing and storing the reports.
 [] - ----=================----
-Found no defects in compiler_warning.cpp
+[MEDIUM] compiler_warning.cpp:3:7: unused variable 'i' [clang-diagnostic-unused-variable]
```

Comment:
Did you intend for this to appear here? Why? I'm sure I can figure out if I stare at it long enough, but this functional test has quite a lot of cryptic changes.

Reply:
Because we test here that `clang-diagnostic-unused-variable` being in the "default" profile is stronger than `-Wno-all` and `-Wno-extra` in the original build command.